addons/website_sale_mondialrelay/static/src/js/checkout.js

```diff
@@ -114,8 +143,10 @@ websiteSaleCheckoutWidget.include({
         rpc('/website_sale_mondialrelay/update_shipping', {
             ...this.lastRelaySelected,
         }).then((o) => {
-            $('#address_on_payment').html(o.address);
-            this.$modal_mondialrelay.modal('hide');
+            if (document.querySelector("#address_on_payment")) {
+                document.querySelector("#address_on_payment").innerHTML = o.address;
```

Comment:
This element is not present on the page when the "Send to this Parcel Point" button is clicked. There seems to be a bug; no new `o_portal_address` element is added under the "Delivery address" title... If we go back by clicking on "Review Order" and then return to the page, the new address appears correctly. But this bug also exists without your changes, @odoo/rd-sales are you aware of this bug? Shouldn't we create a task for it?

Reply:
@vchu-odoo are you aware of this issue ? Can you take a look ? I don't know this logic much, but maybe it was introduced by our recent changes (unless it's been broken for some time already :thinking:)

Reply:
I think that is because of moving delivery methods to checkout instead of payment as `address_on_payment` is on the payment page.

Reply:
@vchu-odoo Do we have to do anything on this PR or you are going to create separate task?

Reply:
Hello @msh-odoo!
The fix for mondialrelay is merged https://github.com/odoo/odoo/pull/181393.
So you can rebase and adapt your changes accordingly.

Reply:
Hello @vchu-odoo,
I’ve rebased the branch and updated it accordingly.

Reply:
Thank you, since this is a task in the Website project, they will review it and I will be pinged when PR is ready for review.